packages/web/src/components/nav/desktop/PlaylistLibrary/CollectionNavItem.tsx

```diff
@@ -211,11 +231,19 @@ export const CollectionNavItem = (props: CollectionNavItemProps) => {
                   <PlaylistUpdateDot />
                 </div>
               ) : null}
-              {name}
+              <Text
+                variant='body'
+                size='s'
+                css={{ maxWidth: '160px' }}
+                ellipses
```

Comment:
can also try `numberOfLines={1}` i think?

Reply:
seems to not be available

Reply:
maybe mobile only, sad